### ISSUE TITLE
CPMenu keyboard navigation doesn't work with submenus

### ISSUE THREAD

**Issue by klaaspieter** [cappuccino/cappuccino#1147]
*\* Steps **
1. Open a menu
2. Open a using the mouse submenu (without selecting any item in the submenu)
3. Start typing

*\* Expected **

The typeahead to search items and select items in the supermenu

*\* Actual **

The typeahead is done in the submenu
